### PR TITLE
DBZ-5321 Use 2.1 images for building UI

### PR DESCRIPTION
**Pull request by ani-sha**

### CODE REVIEW

File: docker-compose-oracle.yml [hunk 1]
```diff
@@ -3,12 +3,12 @@ version: '3'
 services:
   dbzui-zookeeper:
     container_name: dbzui-zookeeper
-    image: debezium/zookeeper:1.8
+    image: debezium/zookeeper:2.1
```
Comment (rk3rn3r) [inline docker-compose-oracle.yml:6]:
```suggestion
    image: quay.io/debezium/zookeeper:2.1
```

Comment (ani-sha) [reply docker-compose-oracle.yml:6]: @rk3rn3r @Naros I have the same error with the `quay.io` images.

**Comment by ani-sha** [comment]
Works fine on my new system, merging.

**Comment by rk3rn3r** [comment]
@ani-sha It looks like you missed some places. For example: https://github.com/ani-sha/debezium-ui/blob/upgrade-to-2.0/backend/src/test/java/io/debezium/configserver/util/Infrastructure.java#L44
Furthermore all docker-compose files should base on the quay.io registry imho.

**Comment by rk3rn3r** [comment]
Could you also please in the future post a link in the first comment / pr description so it is easier to discover the task in Jira?

**Comment by rk3rn3r** [comment]
And also the Jira DBZ-5321 is not about bumping Debezium dependency in the UI to 2.1 but instead about the metrics. Can you please file a separate task for such things in the future?

**Comment by ani-sha** [comment]
@rk3rn3r Sure, I will keep it mind. I have created a new Jira ([DBZ-6173](https://issues.redhat.com/browse/DBZ-6173)) for doing the rest changes. I will link this PR as well.

**Pull request merged**